src/components/BackEndRolePage/Confess.js

```diff
@@ -2,9 +2,18 @@ import React from 'react';
 
 import { SAD } from '../../constants/characterMoods';
 import { Robot } from '../Characters';
+import QuoteBubble from '../QuoteBubble';
+
+// TODO Replace with auto-generated message
+const QUOTE = 'Yesterday I was implementing PUT method for missing RESTful API endpoint.';
```

Comment:
Maybe it's a good time to extract this constant to seperate file e.g. `/constants` as it's polluting code. Of course I'm aware that this implementation will be changed soon... but still 🤷‍♂️

Reply:
I think it serves as good reminder to update it in all required places, since it's such glaring antipattern :)

Reply:
Haha, that's true. Very hard to miss it 🤣